scripts/release/release_pip.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+####
+# Utils
+####
+source ${BASH_SOURCE%/*}/utils.sh
+###
+
+# 1. Get options
+
+## Defaults
+APPLY="false"
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -A|--apply)
+      APPLY="true"
+      shift # past argument
+      ;;
+    -*|--*)
+      echo "Unknown option $1"
+      exit 1
+      ;;
+    *)
+      POSITIONAL_ARGS+=("$1") # save positional arg
+      shift # past argument
+      ;;
+  esac
+done
+
+# 2. Cleaning up
+echo_info "Removing dirs: dist/ and build/"
+rm -rf dist/
+rm -rf build/
+
+# 3. Creating python wheel
+echo_info "Generating python wheel"
+python3 setup.py sdist bdist_wheel
+
+# 3. Upload wheel to pypi
+if [[ $APPLY == "true" ]]; then
+    echo_info "Uploading python wheel"
+    python3 -m twine upload dist/*
```

Comment:
How does the Twine authentication happen for uploading the python wheel?

Reply:
yes, this is one of the important points regarding PyPi.
The one who run this needs to be a maintainer in the bittensor PyPi's project. Eventually i am going to create a system/bot account for that, so no one manually run that. But if a person has to upload this to PyPi, he/shew has to be a maintainer of the project.